Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN make install-frontend
 COPY web/ web/
 RUN make build-frontend
 
-FROM registry.access.redhat.com/ubi9/go-toolset:1.20 as go-builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17 as go-builder
```

Comment:
It seems cachito does not support go 1.22 yet. So we might need to build using `1.21.0` https://redhat-internal.slack.com/archives/C02AX10EQJW/p1723739063535609?thread_ts=1723737266.170519&cid=C02AX10EQJW

Reply:
Based on the [documentation](https://spaces.redhat.com/pages/viewpage.action?pageId=228017926) and that slack thread it looks like this issue isn't exactly the use of 1.22, but rather the lack of a z stream release causing the toolchain to be unable to download the correct version. The resulting [Dockerfile](https://github.com/project-koku/koku-metrics-operator/blob/main/go.mod#L3) from that thread still uses 1.22. I'll update the go.mod to have the z streams needed for the toolchain